### ISSUE TITLE
PoseNet

### ISSUE THREAD

**Issue by Summer-LL** [mks0601/3DMPPE_POSENET_RELEASE#133]
Hello! Thanks for your great working!
In your code, I find the code of RootNet, but cannot find the code of PoseNet. I want to get other key points not root. Running your demo.py, I only got the coordinate of the root. Could you tell me how to get the code of PoseNet or where is it?
Waiting for your reply!
Thanks a lot!

**Comment by mks0601** [comment]
Hi, you can visit here
https://github.com/mks0601/3DMPPE_POSENET_RELEASE

**Comment by Summer-LL** [comment]
Thanks for your answer! I got it in [this link](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/main/model.py) .But a new problem arose that the followed picture when I used your vis.py to draw 3D pose. Your vis.py is [here](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/common/utils/vis.py) . I run demo.py to get 18 key points and then the code is as follows.
Could you tell me how to modify it to draw correctly?
Remark：I run demo.py and not test.py. Because I only test a random image.
<img width="972" alt="image" src="https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/a40ed52c-bf14-4ec4-8dc1-885c7e432ff5">

<img width="462" alt="image" src="https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/37a14a06-0a66-43f0-88dd-6633a05db2af">

**Comment by Summer-LL** [comment]
> Thanks for your answer! I got it in [this link](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/main/model.py) .But a new problem arose that the followed picture when I used your vis.py to draw 3D pose. Your vis.py is [here](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/common/utils/vis.py) . I run demo.py to get 18 key points and then the code is as follows. Could you tell me how to modify it to draw correctly? Remark：I run demo.py and not test.py. Because I only test a random image. <img alt="image" width="972" src="https://user-images.githubusercontent.com/48204499/251329823-a40ed52c-bf14-4ec4-8dc1-885c7e432ff5.png">
> 
> <img alt="image" width="462" src="https://user-images.githubusercontent.com/48204499/251329337-37a14a06-0a66-43f0-88dd-6633a05db2af.png">

**Comment by Summer-LL** [comment]
@mks0601 
Thanks for your answer! I got it in [this link](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/main/model.py) .But a new problem arose that the followed picture when I used your vis.py to draw 3D pose. Your vis.py is [here](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/7df62e283ddc13303f0c4cf3fbd318dd5466eea6/common/utils/vis.py) . I run demo.py to get 18 key points and then the code is as follows.
Could you tell me how to modify it to draw correctly?
Remark：I run demo.py and not test.py. Because I only test a random image.
<img width="972" alt="image" src="https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/a40ed52c-bf14-4ec4-8dc1-885c7e432ff5">

<img width="462" alt="image" src="https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/37a14a06-0a66-43f0-88dd-6633a05db2af">

**Comment by Summer-LL** [comment]
I only set a bounding box, so the number of person is 1.

**Comment by mks0601** [comment]
I'm not sure I got you. Could you clarify?

**Comment by Summer-LL** [comment]
Sorry,I didn't express myself clearly.
I have generated the 3D visual picture. 2D image and 3D image are as follows.  It's obvious that the posture is incorrect in 3D space. What did I do wrong? How should I correct it?
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/419de5df-fcda-4cde-b146-aa8710fa0093)
![image](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/b1be74c7-1349-471c-9671-8030449ba759)

**Comment by Summer-LL** [comment]
Yes, I run [the code](https://github.com/mks0601/3DMPPE_POSENET_RELEASE/blob/master/demo/demo.py) and input my image, bbox_list and root depth which is from RootNet.

**Comment by mks0601** [comment]
I think you need to make tick of xyz have the same interval. Y-axis has an interval of 250, too high than 100 of x-axis and z-axis

**Comment by Summer-LL** [comment]
I got it. I set the range of all three axes to the example diagram you gave. The generated 3D image now looks normal. Thank you for your kind answer.
<img width="518" alt="image" src="https://github.com/mks0601/3DMPPE_POSENET_RELEASE/assets/48204499/adaba911-da9b-41a7-abb4-05da10619b53">
